replica/database.hh

```diff
@@ -589,7 +589,7 @@ private:
     update_sstable_lists_on_off_strategy_completion(sstables::compaction_completion_desc desc);
 
     // Rebuild sstable set, delete input sstables right away, and update row cache and statistics.
-    void on_compaction_completion(sstables::compaction_completion_desc desc);
+    future<> update_main_sstable_list_on_compaction_completion(sstables::compaction_completion_desc desc);
```

Comment:
Absolutely approve of those super long names. Rare and complicated events need long and descriptive names.

Reply:
> Absolutely approve of those super long names. Rare and complicated events need long and descriptive names.

Agreed.